### DISCUSSION TITLE
discussion test

### DISCUSSION THREAD

**Discussion by yourkevin** [yourkevin/NiterForum#12] [General]
discussion test

**Comment by ZhenjiangFan** [comment]
很好的一个项目，谢谢你的开源项目。
有个额外的问题，就是你这个项目名字。这个名字听起来好像 “N word", 在国外有些地方有时候会引起不满。我想大多数情况会没问题。
我过来是谢谢你的开源项目。谢谢！

**Reply by yourkevin** [reply]
感谢你的支持，起名的时候没考虑到这点~
